kartograf/context.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, args):
                 self.args.reproduce += '/'
             self.data_dir = self.args.reproduce
```

Comment:
Oh, I think this is the issue: Here the user input of the path is used and not normalized. I always give the full path which is why this never can up as an issue for me. Now I see different results based on the path caused by the change two lines below. You probably were using a relative path in the input and that's why for you the change below is a fix.